### DISCUSSION TITLE
"Investigating a Star-Formation Shutdown in the Early Universe"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#149] [Astrophysics and Physics]
Astronomers can't find a model that explains the fast star-formation history of a 325 Myr-old galaxy *and* the star-formation shutdown of a 700 Myr-old galaxy.

"Investigating a Star-Formation Shutdown in the Early Universe"
🔗[https://aasnova.org/2023/09/18/investigating-a-star-formation-shutdown-in-the-early-universe/](https://aasnova.org/2023/09/18/investigating-a-star-formation-shutdown-in-the-early-universe/)

**Open Access:** "Quiescent Low-mass Galaxies Observed by JWST in the Epoch of Reionization"
🔗[https://iopscience.iop.org/article/10.3847/2041-8213/acee80](https://iopscience.iop.org/article/10.3847/2041-8213/acee80)

**Comment by bgreenman** [comment]
Right now most mainstream researchers are not looking for alternatives to the big bang's version of galactic evolution, the galaxies are probabily older than the big bang models state

**Reply by RedshiftDrift** [reply]
It's the other way around, astronomers *are* looking for alternative to galaxy evolution models (see the two papers cited above) because they don't want to change the Big Bang model.

**Reply by bgreenman** [reply]
you are correct
